src/frontend/src/store/index.ts

```diff
@@ -86,11 +89,21 @@ export default new Vuex.Store({
         })
         .then((response) => {
           if(response.data.success) {
-            alert("Transaction was successful!");
-            dispatch("ACTION_FETCH_WALLET_AMOUNT");
-          } else if(response.data.err) {
-            alert("Transaction has failed.");
-          }
+            Vue.$toast.success('Transaction has sent!', {
+              message: 'Transaction has sent!',
+              duration: 3000,
+              position: 'top',
+              dismissible: true,
+            });
+            commit("MUTATION_SET_WALLET_AMOUNT", response.data.remaining_balance);
+          } 
+        }, () => {
+          Vue.$toast.error('Transaction has failed.', { 
+            message: 'Transaction has failed.', 
```

Comment:
Can you please write E2E tests that validate the success and fail paths for creating transactions, just to ensure the toast pops up, and the correct one at that, according to if the request succeeds or fails